package.json

```diff
@@ -55,7 +62,9 @@
     "telemetry": "node scripts/telemetry.js",
     "check:lockfile": "node scripts/check-lockfile.js",
     "clean": "node scripts/clean.js",
-    "pre-commit": "node scripts/pre-commit.js"
+    "pre-commit": "node scripts/pre-commit.js",
+    "use-dev": "node scripts/configure-registry.js dev",
```

Comment:
Opinion: I think we should just have people setup an npmrc with both and then there's no need to "switch"